lib/api/application.go

```diff
@@ -21,7 +21,7 @@ func setupContainer(app types.Application, storedir string, setup chan types.Res
 		Name:            app.GetName(),
 		Image:           app.GetDockerImage(),
 		ApplicationPort: app.GetApplicationPort(),
-		ContainerPort:   app.GetContainerPort(),
+		ContainerPort:   app.GetContainerPort(),	
```

Comment:
remove extra space

Reply:
I think ye formatter ne daal diya hoga,
I will remove it

Reply:
iska commit daalde baaki LGTM

Reply:
done